library/ssl_tls12_server.c

```diff
@@ -1131,7 +1131,7 @@ static int ssl_parse_client_hello(mbedtls_ssl_context *ssl)
      * ClientHello layer:
```

Comment:
While we're here, could we change "layer" to "layout"?

Reply:
Yes. I have chagned "layer" to "layout".